app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] When a user signs up the app logs them in at once. Kindly note that when I sign up as a user, I should be required to confirm my email before I can log in. Kindly add the `:confirmable` config to the user devise setup.